app/client/rest/base.ts

```diff
@@ -201,10 +202,18 @@ export default class ClientBase {
         return `${this.getThreadsRoute(userId, teamId)}/${threadId}`;
     }
 
+    getPluginsRoute() {
+        return `${this.urlVersion}/plugins`;
+    }
+
     getAppsProxyRoute() {
         return '/plugins/com.mattermost.apps';
     }
 
+    getCallsRoute() {
```

Comment:
Possibly... But I think it might be simpler to make it a straight function call like this (for when the id changes in the future, and for the calling code). I'm 1/5 on it though.